index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio - Mobile Version</title>
+    </head>
+    <body>
+        <main>
+            <div class="header-section">
+                <header class="header">
+                    <nav class="nav">
+                        <img src="assets/my-logo.png" alt="My Logo" />
+                        <div class="hamburger-menu">
+                            <div class="top-line"></div>
+                            <div class="middle-line"></div>
+                            <div class="bottom-line"></div>
+                        </div>
+                    </nav>
+                </header>
+                <div class="profile-section">
+                    <p class="about-header">
+                        I’m Olawale Glad to see you!
+                    </p>
+                    <p class="about-body">
+                        I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! If you
+                        like what you see and have a project you need coded, don’t hestiate to 
+                        contact me.
+                    </p>
+                    <a href="#" class="link">LET'S CONNECT</a>
+                    <ul class="socials">
+                        <li class="social-item">
+                            <a href="#">
+                                <img src="assets/twitter.png" class="social-img" alt="twitter" />
+                            </a>
+                        </li>
+                        <li class="social-item">
+                            <a href="#">
+                                <img src="assets/linkedin.png" class="linkedin" alt="linkedin" />
+                            </a>
+                        </li>
+                        <li class="social-item">
+                            <a href="#">
+                                <img src="assets/medium.png" class="social-img" alt="medium" />
+                            </a>
+                        </li>
+                        <li class="social-item">
+                            <a href="#">
+                                <img src="assets/github.png" class="social-img" alt="github" />
+                            </a>
+                        </li>
+                        <li class="social-item">
+                            <a href="#">
+                                <img src="assets/finger.png" class="social-img" alt="finger" />
+                            </a>
+                        </li>
+                    </ul>
+                </div>
```

Comment:
- [ ] Since this is a new section, the container should therefore be a `section` tag. Using HTML5 semantic tags is one of  HTML & CSS best practices